Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs

```diff
@@ -246,6 +250,16 @@ void UpdatePadding()
 				(int)(Context.ToPixels(padding.Bottom + adjustment) + defaultPadding.Bottom));
 		}
 
+		void UpdateLineBreakMode()
+		{
+			AppCompatButton view = View;
+			if (view == null || _element == null)
+				return;
+
+			view.SetLineBreakMode(_element);
+			view.SetAllCaps(true);
```

Comment:
??

Reply:
@samhouts, sorry I didn't understand

Reply:
Are we supposed to be forcing AllCaps here? I didn't see this in the original version.

Reply:
I'm sorry for the late reply... Well if we don't use that, the text in the button will not follow the actual behavior for text in the Android's button and text will not be all caps.

Reply:
@samhouts, I'm sorry for the late reply... Well if we don't use that, the text in the button will not follow the actual behavior for text in the Android's button and text will not be all caps.

Reply:
@pictos What causes it to not be all caps anymore?

Reply:
@samhouts I don't know why that is, but I can try to find out (if it's needed). Just to exemplify, if I remove this line, the result is:
![image](https://user-images.githubusercontent.com/20712372/77131842-f983ca80-6a3b-11ea-88be-1e7d83fad8ea.png)

Reply:
@pictos I think it's work investigating why that is different. Thanks!